internal/gotty/client.go

```diff
@@ -0,0 +1,208 @@
+package gotty
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/containerd/console"
+	"github.com/gorilla/websocket"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+// Scaleway GoTTY websocket endpoint
+var wsURLs = map[scw.Zone]string{
```

Comment:
Not sure which is better between `tty.scaleway.com` and `tty-<region>.scaleway.com` :thinking:

Reply:
tty.scaleway.com do not work for AMS

Reply:
:scream: :heavy_check_mark: